packages/design-system/src/Radio/Radio.styles.tsx

```diff
@@ -24,7 +31,7 @@ const BasicStyles = css`
     position: absolute;
     width: var(--ads-v2-spaces-5);
     height: var(--ads-v2-spaces-5);
-    background-color: var(--radio-color-background);
+    background-color: var(--ads-v2-colors-control-radio-tack-default-bg);
```

Comment:
Can we have the category token be consumed by the component token and only the component token be used by the inner css of the component? Especially when both types of tokens are present in a file.